support.html

```diff
@@ -43,6 +43,13 @@ <h2><a href="{{ redHatLink }}">Red Hat</a></h2>
                 </ul>
             </p>
         </section>
+
+        <section class="col-12_md-12">
+          <h2><a href="{{ cloudRaftLink }}">CloudRaft</a></h2>
+          <p>
+              CloudRaft is an AI and Cloud Native Consulting company providing support for Rook and other projects such as Kubernetes, Ceph and other open source projects.
+          </p>
```

Comment:
I don't see any info about Rook on your website. The single line you've provided is very high level. Could you provide one or two bullet points that describe more about how Rook is supported by CloudRaft? It will help Rook users know to reach out for more info.